.tekton/coo-fbc-v4-14-pull-request.yaml

```diff
@@ -224,7 +247,7 @@ spec:
         value: $(params.build-image-index)
       - name: IMAGES
         value:
-        - $(tasks.build-container.results.IMAGE_URL)@$(tasks.build-container.results.IMAGE_DIGEST)
+        - $(tasks.build-images.results.IMAGE_REF[*])
       runAfter:
       - build-container
```

Comment:
this should be build-images. note that build-container no longer exists.

Reply:
done